common4j/src/main/com/microsoft/identity/common/java/eststelemetry/EstsTelemetry.java

```diff
@@ -56,11 +57,24 @@ public class EstsTelemetry {
     private static final String LAST_REQUEST_TELEMETRY_STORAGE_FILE =
             "com.microsoft.identity.client.last_request_telemetry";
```

Comment:
Noob question: Is there no benefit in having strings in a dedicated resource file?

Reply:
I don't think there is much benefit, and furthermore, I think resource file is an "Android" concept and not something that can be used in 4j projects.